samples/snippets/csharp/tutorials/inheritance/shape.cs

```diff
@@ -64,7 +64,7 @@ public Circle(double radius)
       Radius = radius;
    } 
 
-   public override double Area => Math.Round(Math.Pow(Math.PI * Radius, 2), 2); 
+   public override double Area => Math.Round(Math.PI * Radius * Radius, 2); 
```

Comment:
I'm new to C# and OOP in general so it took me a moment to understand `Math.Pow` and what it was doing. For myself `Math.PI * Radius * Radius` is immediately clearer than `Math.PI * Math.Pow(Radius, 2)`.
`Pow` is used above so it would make sense to keep with the flow. It also provides a means of exposing `Math.Pow` to those that aren't aware of it.